src/lib/rnp.cpp

```diff
@@ -70,6 +70,13 @@
         RNP_LOG_FD(fp, __VA_ARGS__); \
     } while (0)
 
+#if defined(RNP_EXPERIMENTAL_CRYPTO_REFRESH) && !defined(ENABLE_CRYPTO_REFRESH)
+#error "Invalid defines combination."
```

Comment:
(This is a suggestion to consider)
Pls refer to the note about CRYPTO_REFRESH flags